src/test/java/seedu/address/commons/util/DateUtilTest.java

```diff
@@ -0,0 +1,30 @@
+package seedu.address.commons.util;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.util.Calendar;
+import java.util.Date;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Tests DateUtil.
+ */
+class DateUtilTest {
+
+    @Test
+    public void parseDate_validDate() throws ParseException {
```

Comment:
The convention for test names is `methodName_testScenario_expectedResult`. expectedResult seems to be missing.